plugins/_collections.js

```diff
@@ -381,11 +381,35 @@ exports.attrsGroupsDefaults = {
   },
 };
 
+/**
+ * @type {Record<string, { safe?: Set<string>, unsafe?: Set<string> }>}
+ * @see https://www.w3.org/TR/SVG11/intro.html#Definitions
+ */
+exports.attrsGroupsDeprecated = {
+  animationAttributeTarget: { unsafe: new Set(['attributeType']) },
+  conditionalProcessing: { unsafe: new Set(['requiredFeatures']) },
+  core: { unsafe: new Set(['xml:base', 'xml:lang', 'xml:space']) },
+  presentation: {
```

Comment:
Presentation attributes can be used in styles as well. We should probably be removing them too.

Could you please add a check where if the attribute being removed is a presentation attribute, also parse the `style` attribute and remove instances from there too. If the style attribute becomes empty, delete the style attribute entirely.

I worked on something like this recently in the `cleanupEnableBackground` plugin, so you can use it as a reference:

* https://github.com/svg/svgo/pull/1866/files?short_path=3741d79#diff-0805bdcf601afdd4c7674c3595fee8ea9da61677f6f36a4fdff25bd3f39fb840R45-R68

Basically, if the node defines the `style` attribute, parse the styles and if it's a `DeclarationList`, iterate the list and removing unwanted styles. Finally you can generate the new styles with `csstree.generate` and save it back, or delete the attribute if it's become empty.

Important to note that the same attribute can redundantly defined more than once in CSS, so it's worth looping through all styles and not exit early if the attribute is found.

It could be worth updating the stylesheet (`<style>` tags) too.